requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit
```

Comment:
Use full version numbers here so we get the same version for each run. If we want to use requirements.txt rather then Pipfile (pipenv) of pyproject.toml (poetry), then we probably should use pip-tools to get the usual unversioned+lock files combination.

Reply:
fixed, chose to use `.tmpl.`